rfc/0010-rancher-integration-policies.md

```diff
@@ -0,0 +1,188 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Rancher integration of Kubewarden Policies  |
+| Start Date   | 2022-08-22                       |
+| Category     | Development                      |
+| RFC PR       | https://github.com/kubewarden/rfc/pull/11  |
+| State        | **ACCEPTED**                     |
+
+
+# Summary
+[summary]: #summary
+
+This RFC explains how the Kubewarden policies, either owned by the Kubewarden
+team or 3rd party ones, can be integrated in Rancher Explorer.
+
+For policies owned by the Kubewarden team it proposes building Rancher Helm charts
+that will be published under https://github.com/rancher/charts.
+
+For 3rd party policies not owned by the Kubewarden team,
+it proposes using the Artifact Hub API to poll for the metadata stored in
+`policy/artifacthub-pkg.yml`.
+
+
+# Motivation
+[motivation]: #motivation
+
+Problems to solve:
+
+- Obtain the list of available policies regardless of where they come from, and
+  their metadata to show in Rancher Explorer UI.
+- Allow for groups of policies to be installed.
+- Implement it efficiently and with low development maintenance costs.
+- Allow for airgap installations.
+
+## Examples / User Stories
+[examples]: #examples
+
+1. As a user, I want to install one Kubewarden policy owned by the
+   Kubewarden/Rancher team via Rancher Explorer UI.
+2. As a Kubewarden developer, I want to allow users to deploy a collection of related policies
+   in a simple way. For example: "PSP best of", "Security best practices",...
+3. As a user, I want to install 3rd party Kubewarden policies via Rancher
+   Explorer UI.
+4. As a Kubewarden developer, I want to release a new Kubewarden chart/policy so
+   it can be installed via Rancher Explorer UI.
+5. As a 3rd party developer, I want to release a policy on Artifact Hub so it can
+   be discovered and installed via Rancher Explorer UI.
+
+# Detailed design
+[design]: #detailed-design
+
+## For policies owned by the Kubewarden team
+
+Follow the process listed in [RFC-9, Rancher integration of Kubewarden
+charts](./0009-rancher-integration-charts.md).
+
+Carry the Helm chart code for the policy in a `rancher-chart` folder in the
+policy repo.
+
+Still, even if the chart will not be released to users, the repository can be
+consumed in Rancher UI (using git URL and branch) for testing the chart itself.
```

Comment:
Did you have time to look into the option we were talking: adding these policies helm charts to the https://github.com/kubewarden/helm-charts repository?

Reply:
Yes, I'm iterating on it. I have opened https://github.com/kubewarden/helm-charts/pull/130/files.

Argh, sorry, amended the intro in the past, but not the rest of the document. Updated it in https://github.com/kubewarden/rfc/pull/12/commits/f703d00905b4540e295e2f222be4012c0804e54d